lib/fry.rb

```diff
@@ -0,0 +1,12 @@
+module PlanetExpress
+  class Fry
+
+    attr_accessor :name, :thirst
+
+    def initialize(game)
+      @name = "Fry"
+      @thirst = 50
```

Comment:
You might be able to put a constant for this starting variable in the module, and then each class that is inside this module could utilize that constant.